poc/TestOfTestFrameworkByReference/NFUnitTestByReference.nfproj

```diff
@@ -37,17 +38,16 @@
     <ProjectReference Include="..\..\source\UnitTestLauncher\nanoFramework.UnitTestLauncher.nfproj" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```

Comment:
app config are bad :-D can you remove it?

Reply:
Removed.

Reply:
BTW why don't you like app.configs? What's wrong with them?

Reply:
> BTW why don't you like app.configs? What's wrong with them?

Is not that we don't like them (they're very useful). The thing is that this is _nano_ the type resolution can't handle the assembly remapping just like the big framework. That would be too "expensive". 😉 

By adding the app.config the compiler gets happy, but you'll hit most likely type resolution errors when deploying.